tensorboard/components/tf_data_selector/tf-data-select-row.html

```diff
@@ -0,0 +1,82 @@
+<!--
+@license
+Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../paper-dropdown-menu/paper-dropdown-menu.html">
+<link rel="import" href="../paper-item/paper-item.html">
+<link rel="import" href="../paper-listbox/paper-listbox.html">
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../tf-backend/tf-backend.html">
+<link rel="import" href="../tf-color-scale/tf-color-scale.html">
+<link rel="import" href="../tf-dashboard-common/scrollbar-style.html">
+<link rel="import" href="../tf-dashboard-common/tf-filterable-checkbox-dropdown.html">
+
+<!--
+tf-data-selector creates a set of checkboxes to display which runs are
+selected. It also displays tooltips.
+
+Properties in: none.
+Properties out:
+- selectedRuns: The array of run names that are currently checked by the user.
+
+-->
+<dom-module id="tf-data-select-row">
+  <template>
+    <template is="dom-if" if="[[_experiments.length]]">
+      <span>
+        <paper-dropdown-menu label="Experiment" horizontal-align="left">
```

Comment:
Any reason why the experiment dropdown isn't also a tf-filterable-checkbox-dropdown so that it can be regex-filtered?  Is that just not yet implemented?

Reply:
I started off that way but experiment dropdown is not a multi-selectable. Therefore, I thought, checkbox, regex filter, or the toggle all button are incorrect and decided to use the default dropdown menu (you can still type out name of the experiment (like real `<select>`) to jump to it.